Lib/test/test_capi/test_opt.py

```diff
@@ -2093,6 +2093,19 @@ def testfunc(n):
         self.assertNotIn("_POP_CALL_ONE_LOAD_CONST_INLINE_BORROW", uops)
         self.assertNotIn("_POP_CALL_TWO_LOAD_CONST_INLINE_BORROW", uops)
 
+    def test_call_isinstance_guards_pop_top(self):
```

Comment:
Can't leave a comment on that line, but inside `test_call_isinstance_guards_removed` we should include the new opcode `_SWAP_CALL_TWO_LOAD_CONST_INLINE_BORROW` to ensure `remove_unneeded_uops` is still working as expected (I think it should be since you also updated `op_without_push`, but just to be sure :))